### DISCUSSION TITLE
docker hello-world permission denied

### DISCUSSION THREAD

**Discussion by WenqiJiang** [milvus-io/milvus#5110] [Q&A and General discussion]
Dear Milvus team,

I followed the installation guide of the hello world: https://milvus.io/docs/v1.0.0/milvus_docker-cpu.md . But when starting the container, I got a permission deny problem on mkdir the database / log / conf directories. I already chmod the milvus folder that all users can read and write...

![image](https://user-images.githubusercontent.com/42897680/116894729-ae724a00-ac32-11eb-8d96-9b86e0580dad.png)

![image](https://user-images.githubusercontent.com/42897680/116894804-c8139180-ac32-11eb-8930-e64b67e97805.png)

Could you help to figure out what is going wrong? 

Thanks a lot

**Comment by yhmo** [comment]
I guess you run the docker container incorrectly. Some path mapping is incorrect.
Assume you want to specify **/home/wejiang/milvus** as milvus data folder.
Firstly, you need to download the **https://raw.githubusercontent.com/milvus-io/milvus/v1.0.0/core/conf/demo/server_config.yaml** to **/home/wejiang/milvus/conf**

Secondly, in the **/home/wejiang/milvus/conf/server_config.yaml**, make sure all the path is point to inside path:
```
storage:
  path: /var/lib/milvus
wal:
  path: /var/lib/milvus/wal
logs:
  path: /var/lib/milvus/logs
```

Third, run docker container by mapping paths from inside to outside:
`docker run -it -p 19530:19530 -v /home/wejiang/milvus/db:/var/lib/milvus/db -v /home/wejiang/milvus/conf:/var/lib/milvus/conf -v /home/wejiang/milvus/logs:/var/lib/milvus/logs milvusdb/milvus:1.0.0-cpu-d030521-1ea92e`
